client/src/use/useTextLayer.js

```diff
@@ -9,8 +9,8 @@ export default function useTextLayer({
   const vuetify = inject('vuetify');
 
   const textData = computed(() => {
-    if (!filteredDetections.value.length) {
-      return null;
+    if (filteredDetections.value.length === 0) {
+      return [];
```

Comment:
You changed this from a falsey to a truthy value, causing the template `v-if` to change meaning.